k8s/helm/values.yaml

```diff
@@ -17,10 +17,9 @@ service:
 
 ingress:
   enabled: true
-  annotations:
-    kubernetes.io/ingress.global-static-ip-name: survey-runner
-  tls:
-    - secretName: tls-cert
+  # Use a managedCertificateName when terraform is used to create a managed certificate
+  # This should be left undefined when using a kubernetes secret for the certificate
```

Comment:
Minor, but this shouldn't mention Terraform as it doesn't matter what created the managed certificate.